src/roadmap/release-checklist.md

```diff
@@ -34,47 +36,92 @@ A release consists of the following artifacts:
 images referenced above. For example, if `$WORKFLOW_RELEASE` is `2.0.0-rc1`, the new chart would
 be in a new directory called `workflow-rc1`.
 
-# Step 1: Create New Helm Classic Charts
+# Step 1: Cut repo branches and push image tags
 
-First, export necessary values for `WORKFLOW_RELEASE` and `WORKFLOW_RELEASE_SHORT`:
-```
-export WORKFLOW_RELEASE=<full release name>
-export WORKFLOW_RELEASE_SHORT=<short form of above>
-```
+(For actions below, [sgoings/deis-workflow-group](https://github.com/sgoings/deis-workflow-group) was used)
+
+1. Once the release milestone is cleared of tickets in the workflow component repos, the release branches can be cut.  
+
+  If only a particular repo is ready, navigate to said repo and:
+  ```
+  git checkout master && git pull origin master
+  git checkout -b release-$WORKFLOW_RELEASE && git push origin release-$WORKFLOW_RELEASE
+  ```
+  Otherwise, for bulk-cutting all repos at the same time:
+  ```
+  make git-update # point all repos to latest master commits
+  BRANCH="release-$WORKFLOW_RELEASE" NEW="true" make git-checkout-branch
+  BRANCH="release-$WORKFLOW_RELEASE" make git-push-branch #(can use DRY_RUN=true)
+  ```
+
+2. tag and push docker images to 'staging' `deisci` org
+  ```
+   TAG="${WORKFLOW_RELEASE}" ORG="deisci" make docker-tag #(can use DRY_RUN=true)
+   TAG="${WORKFLOW_RELEASE}" ORG="deisci" make docker-push #(can use DRY_RUN=true)
+  ```
```

Comment:
I the preceding lines (52 - 61), it's not clear which project contains these make targets. In fact, looking around, I cannot even find them. More clarity would be better here.